script/main.py

```diff
@@ -25,14 +25,17 @@ def init_argparser():
 
 def find_records(people_json):
   record_names_phone = []
-  record_names_with_dob =[]
+  record_names_picture = []
```

Comment:
The logic for removing the records with pictures should be in the `remove_record_with_picture_from_json` function

Reply:
Searching and removing a record should be in one function?

Reply:
You are not searching here.
You are only removing records given some predicate (rule)
Search would be if you want to fins some records and at the same time you will also remove some of them.

Reply:
Was it about something like this?

Reply:
Yes that's better.

BTW I think it would be better to split commands in different functions. so main() will not be overloaded.
something like this:
```python
if args.operation == 'init':  
    init()
elif args.operation == 'percentage':
    percentage()
```

Reply:
Do these functions move inside init ()?
``` 
    create_new_record_with_dob_in_json(dob_records)
    removes_special_characters_from_phone_and_cell_numbers(phone_records)
    remove_record_with_picture_from_json(people_json)
```

Reply:
This question is part pf the task :-)

Reply:
it's done ;)